whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+a shallow-wide survey for weak lensing science.  A strategy question for LSST is
+whether to go wide first and then deep, or the reverse.
+
+There are actually several drivers for depth over area, given fixed observing
+time and camera+telescope etendue.  A deep observing strategy maximizes the
+cosmological signal-to-noise ratio either by maximizing the signal or minimizing
+the noise.
+
+First, it boosts the amplitude of the cosmic shear signals due to the increased
+amplitude of the lensing kernel.  Given the same lens mass, the amplitude of
+lensing signals is a simple function of the two distances: observer-to-lens
+versus lens-to-source. For example, when a lens is at z = 0.5, the shear for a
+source at z = 1.1 is nearly twice that at z = 0.7, leading to a factor of 4
+ratio in correlation function amplitudes.  Thus, a deep survey can take
+advantage of the geometric effect of gravitational lensing more efficiently.
+
+Second, it enables a longer redshift baseline to break parameter degeneracies.
+For example, shallower surveys with wider area are not efficient in shrinking
+the length of the ``banana'' in the $\Omega_M$-$\sigma_8$ plane because of the
+$\sigma_8 \Omega_M^\alpha = \mathrm{const}$ degeneracy.  The deep strategy
+compensates for the loss of volume due to a reduced area by extending the volume
+along the line of sight.
+
+Third, deep surveys provide more useful galaxies per area.  This merit does not
+entirely overlap with the second point.  In addition to new sources at higher
+redshift, the fainter limiting magnitude enables detection of fainter galaxies
+at a given redshift and better signal-to-noise ratio.  Needless to say, the
+increased source density reduces the impact of shape noise caused by intrinsic
+ellipticity dispersion.
+
+Fourth, it mitigates the effects of intrinsic alignments (IA), an important
+theoretical systematic in precision cosmic shear.  Current studies
+\citep{Heymans2013} indicate that IA effects are dominated by luminous red
+galaxies (LRGs).  Because a deep survey can access fainter galaxies, the net IA
+systematic decreases because the fraction of the LRGs decreases at fainter
+limiting magnitude.  Using the approach of \citet{Joachimi2011}, at z = 0.5 the
+amplitude of the IA power spectrum is reduced by a factor of two for an increase
+in the limiting r magnitude from 24 to 27 mag.
+
+For cosmic shear the volume at 5-60 arcmin scales and over a wide range of
+redshift is most important. Once a fair sampling of “cosmic variance” is
+achieved the depth matters most, because of the z-width of the lensing kernel.
+With fixed survey duration on a camera+telescope of fixed etendue, it is better
+to prioritize depth in order to maximize the weak lensing cosmological
+signal-to-noise ratio.  In particular, for the LSST survey it would be helpful
+to have several widely spaced deep drilling fields done early and deeper than
+the main survey, in order to explore detailed observing strategies --
```

Comment:
It would also provide some less noisy priors on the distribution of galaxy properties, for those shear estimation methods that need them (e.g. Bernstein & Armstrong).